#
# This "please upgrade" message would drive users crazy. Bug #819703
#
Index: xscreensaver-git/driver/prefs.c
===================================================================
--- xscreensaver-git.orig/driver/prefsw.c
+++ xscreensaver-git/driver/prefsw.c
@@ -1743,6 +1743,8 @@ senesculent_p (void)
      of date.
   */
 
+  return 0;
+
   time_t now = time ((time_t *) 0);				/*   d   */
   struct tm *tm = localtime (&now);				/*   o   */
   const char *s = screensaver_id;				/*   n   */
--- xscreensaver-git.orig/driver/demo-Gtk.c	2021-05-13 04:18:53.475000000 +0200
+++ xscreensaver-git/driver/demo-Gtk.c	2021-05-13 04:19:07.637000000 +0200
@@ -5315,7 +5315,7 @@ main (int argc, char **argv)
     the_network_is_not_the_computer (s);
 
 
-  if (time ((time_t *) 0) - XSCREENSAVER_RELEASED > 60*60*24*30*17)
+  if (0 && time ((time_t *) 0) - XSCREENSAVER_RELEASED > 60*60*24*30*17)
     warning_dialog (s->toplevel_widget,
       _("Warning:\n\n"
         "This version of xscreensaver is VERY OLD!\n"
diff --git a/driver/dialog.c b/driver/dialog.c
index 9a8173f..274f775 100644
--- a/driver/dialog.c
+++ b/driver/dialog.c
@@ -1283,7 +1283,7 @@ window_draw (window_state *ws)
   lines[i].align = CENTER;
   i++;
 
-  if (time ((time_t *) 0) - XSCREENSAVER_RELEASED > 60*60*24*30*17)
+  if (0 && time ((time_t *) 0) - XSCREENSAVER_RELEASED > 60*60*24*30*17)
     {
       lines[i].text  = _("Update available!\nThis version is very old.\n");
       lines[i].font  = ws->error_font;
