From a1398d36fb6cc138611b5b7c1a3383be948c59a1 Mon Sep 17 00:00:00 2001
From: unman <unman@thirdeyesecurity.org>
Date: Tue, 19 Oct 2021 02:29:50 +0000
Subject: [PATCH] utils

---
 utils/version.h           | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/utils/version.h b/utils/version.h
index 28347f3..420919e 100644
--- a/utils/version.h
+++ b/utils/version.h
@@ -1,4 +1,4 @@
 static const char screensaver_id[] =
-	"@(#)xscreensaver 6.07 (29-Aug-2023), by Jamie Zawinski (jwz@jwz.org)";
+	"@(#)xscreensaver 6.07 (29-Aug-2023), by Jamie Zawinski";
 #define XSCREENSAVER_VERSION "6.07"
 #define XSCREENSAVER_RELEASED 1693335600
